package.json

```diff
@@ -80,5 +80,11 @@
         "never"
       ]
     }
+  },
+  "babel": {
+    "presets": [
+      "es2015",
+      "react"
+    ]
```

Comment:
or we could have all the configs into package.json and that's it ... what do you think @nickbalestra ?